lib/adb.js

```diff
@@ -3,6 +3,7 @@ import os from 'os';
 import path from 'path';
 import methods from './tools/index.js';
 import { rootDir } from './helpers';
+import DEFAULT_ADB_EXEC_TIMEOUT from './tools/system-calls';
```

Comment:
most likely it should be imported being enclosed in curly brackets:

`import { DEFAULT_ADB_EXEC_TIMEOUT } from './tools/system-calls';`